Application/EdFi.Ods.Common/Providers/Criteria/AggregateRootCriteriaProviderHelpers.cs

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Linq;
+using EdFi.Ods.Common.Specifications;
+
+namespace EdFi.Ods.Common.Providers.Criteria;
+
+internal static class AggregateRootCriteriaProviderHelpers
```

Comment:
These members moved out of the `AggregateRootCriteriaProviderBase<T>` class because it's generic, and we only need a single instance of these, not one per entity.